pkg/quarterdeck/db/models/users.go

```diff
@@ -1097,7 +1097,7 @@ func (u *User) RemoveOrganization(ctx context.Context, orgID any, force bool) (k
 
 const (
 	insertRevokedUserKeysSQL = `INSERT INTO revoked_api_keys
-		SELECT k.id, k.key_id, k.name, k.organization_id, k.project_id, k.created_by, k.source, k.user_agent, k.last_used, p.perms, k.created, k.modified
+		SELECT k.id, k.key_id, k.name, k.organization_id, k.project_id, k.created_by, k.source, k.user_agent, k.last_used, p.perms, k.created, k.modified, :revoked AS revoked
```

Comment:
clever solution to revoke the keys, you could probably also just do `, datetime('now') as revoked` -- but this lets us have a lot more control of the timestamp I think.